src/config_file.c

```diff
@@ -541,6 +590,21 @@ RaucSlot *find_slot_by_bootname(GHashTable *slots, const gchar *bootname)
 	return slot;
 }
 
+static RaucSlot *find_slot_by_name(GHashTable *slots, const gchar *name)
```

Comment:
There is no benefit of splitting this up if it is not used. We must not add unnecessary abstractions.

Reply:
Trying to be consistent with how find_config_slot_by_device and find_slot_by_device are split.  The benefit would be someone who sees find_slot_by_device() could look for and find find_slot_by_name.